azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -24,6 +24,7 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.codehaus.jettison.json.JSONObject;
```

Comment:
We also use the gson library. Is there any reason to use this json library over gson?

Reply:
Thanks for the suggestion! I didn't realize that gson was so convenient to use. I'll push a new commit with the simplified code.